src/FullArticle.jsx

```diff
@@ -16,7 +16,7 @@ const FullArticle = () => {
         }).catch((err) => {
             console.log(err)
         })
-    }, [article_id])
+    }, [article_id], setIsLoading(true))
```

Comment:
This is causing an infinite re-render at the moment and so I can't view the page. The setIsLoading should happen inside of the function. First, before the axios call to indicate were waiting, then back to false at the end of the async chain. Please fix this before continuing. This state should also be false by default.

Reply:
apologies